formats/dot/internal/errors/errors.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// generated by gocc; DO NOT EDIT.
+// Code generated by gocc; DO NOT EDIT.
```

Comment:
Off topic, but the content of blas/cgo/level1doubleBench_auto_test.go seems a bit weird. Is the comment supposed to contain `\n` characters?

From https://github.com/gonum/blas/commit/0285886d824b705f344567a692d06cf96179ad67#diff-c6db70f548cdc8ca223ae67f49e9d561
```go
// Code generated by \"go run github.com/gonum/blas/testblas/benchautogen/autogen_bench_level1double.go\"; DO NOT EDIT.\n
```


https://github.com/gonum/blas/commit/0285886d824b705f344567a692d06cf96179ad67#diff-c6db70f548cdc8ca223ae67f49e9d561